taxi/forms.py

```diff
@@ -0,0 +1,53 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm, get_user_model
+
+from taxi.models import Car, Driver
+
+
+class DriverLicenseValidateMixin:
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != 8:
+            raise forms.ValidationError(
+                "License number must be 8 characters long"
+            )
+        if (
+            license_number[:3].upper() != license_number[:3]
+            or license_number[:3].isalpha() is False
+        ):
+            raise forms.ValidationError("License number must be alphanumeric")
+        if license_number[3:].isdigit() is False:
+            raise forms.ValidationError("License number must be alphanumeric")

```

Comment:
The error message 'License number must be alphanumeric' is misleading here. The check is specifically for the numeric part of the license number. Consider changing the message to something like 'The last 5 characters of the license number must be digits.'